joss-paper/paper.bib

```diff
@@ -0,0 +1,145 @@
+  keywords = {Python, Just-in-time compiler, Benchmark},
+}
+
+@misc{NUMBA,
+  author = {{T. Olifant et al.}},
+  title  = {Numba},
+  howpublished    = {\url{http://numba.pydata.org}},
+}
+
+@InProceedings{PyPy2009,
+  author    = {Bolz, Carl Friedrich and Cuni, Antonio and Fijalkowski, Maciej and Rigo, Armin},
+  title     = {{Tracing the meta-level: PyPy’s tracing JIT compiler}},
+  booktitle = {Proceedings of the 4th Workshop on the Implementation, Compilation, Optimization of Object-Oriented Languages and Programming Systems},
+  year      = {2009},
+  series    = {ICOOOLPS ’09},
+  publisher = {ACM},
+  location  = {Genova, Italy},
+  isbn      = {9781605585413},
+  pages     = {18–25},
+  doi       = {10.1145/1565824.1565827},
+  url       = {https://doi.org/10.1145/1565824.1565827},
+  numpages  = {8},
+}
+
+@InProceedings{Haskell2007,
+  author    = {Hudak, Paul and Hughes, John and Peyton Jones, Simon and Wadler, Philip},
+  title     = {A History of Haskell: Being Lazy with Class},
+  booktitle = {Proceedings of the Third ACM SIGPLAN Conference on History of Programming Languages},
+  year      = {2007},
+  series    = {HOPL III},
+  publisher = {Association for Computing Machinery},
+  location  = {San Diego, California},
+  isbn      = {9781595937667},
+  pages     = {12–1–12–55},
+  doi       = {10.1145/1238844.1238856},
+  url       = {https://doi.org/10.1145/1238844.1238856},
+  address   = {New York, NY, USA},
+  numpages  = {55},
+}
+
+@Article{Sundblad1971,
+  author  = {Sundblad, Y.},
+  title   = {{The Ackermann function. A theoretical, computational, and formula manipulative study.}},
+  journal = {BIT},
+  year    = {1971},
+  volume  = {11},
+  pages   = {107-119},
+  doi     = {10.1007/BF01935330},
+}
+
+@Article{Ackermann1928,
+  author  = {Ackermann, W.},
+  title   = {{Zum Hilbertschen Aufbau der reellen Zahlen}},
+  journal = {Math. Ann.},
+  year    = {1928},
+  volume  = {99},
+  pages   = {118-133},
+  doi     = {10.1007/BF01459088},
+}
+
+@misc{python_operators,
+  author = {{Python Operators}},
+  howpublished = {\url{https://docs.python.org/3/library/operator.html}},
+  }
+
+@misc{python_builtins,
+  author = {{Python Built-in Functions}},
+  howpublished = {\url{https://docs.python.org/3/library/functions.html}},
+  }
+
+@misc{python_standard_library,
+  author = {{Python Standard Library}},
+  howpublished = {\url{https://docs.python.org/3/library/}},
+  }
+  
+@misc{python_math_library,
+  author = {{Python Math Library}},
+  howpublished = {\url{https://docs.python.org/3/library/math.html}},
+  }
```

Comment:
I think all these references can be removed

Reply:
I agree